### ISSUE TITLE
[Remind] pytorch1.0: two errors

### ISSUE THREAD

**Issue by HymEric** [jiecaoyu/XNOR-Net-PyTorch#65]
In pytorch1.0: 
1. When run _python main.py_ in MNIST folder, it has an error . We should change data type  in  _LeNet-5.py_ and shoule be
line45: self.bn = nn.BatchNorm2d(int(input_channels/size), eps=1e-4, momentum=0.1, affine=True)
2. And there are two places needed to change in the _main.py_
line 51:               100. * batch_idx / len(train_loader), loss.item()))

line66:         test_loss += criterion(output, target).item()

**Comment by jiecaoyu** [comment]
Hi @HymEric . The code is updated. Thanks!
